--- phing-2.4.7.1/usr/share/pear/phing/tasks/ext/PhpLintTask.php~	2011-09-29 18:41:57.000000000 +0300
+++ phing-2.4.7.1/usr/share/pear/phing/tasks/ext/PhpLintTask.php	2011-09-29 18:42:11.258207228 +0300
@@ -210,7 +210,7 @@
         $command = $this->Interpreter == ''
             ? 'php'
             : $this->Interpreter;
-        $command .= ' -l ';
+        $command .= ' -n -l ';
         if(file_exists($file)) {
             if(is_readable($file)) {
                 if ($this->cache)
